ansible/group_vars/gelato-ha.yml

```diff
@@ -22,6 +22,9 @@ dummy:
 # GENERAL #
 ###########
 
+# User permitted to access the Kubernetes Cluster to deploy the multicloud services.
+k8s_user: K8S_USER 
```

Comment:
any default user if K8S_USER not specified?

Reply:
It depends on the user configured on K8s cluster.. so not sure if any default user can be added here.

Reply:
if its not a must have, can it be done in a follow up PR.